File: src/server/TRB.Server.Application/api/Requestbodies/UserProfileUpdateRequestDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TRB.Server.Application.api.requestbodies
{
    public class UserProfileUpdateRequestDto
    {
        public DateTime? BirthDate { get; set; }
        public string? Gender { get; set; }
        public string? Address { get; set; }
        public string? Nickname { get; set; }
        public string? ProfileImage { get; set; }
    }
}

File: src/client/src/application/api.requestbodies/UserProfileUpdateRequestDto.ts
export interface UserProfileUpdateRequestDto {
  birthDate?: string;       // yyyy-MM-dd 형식 (선택사항)
  gender?: string;          // "M" 또는 "F"
  address?: string;
  nickname?: string;
  profileImage?: string;    // 경로 or URL
}